geth/utils/encoding.py

```diff
@@ -1,23 +1,27 @@
 import codecs
+from typing import (
+    Any,
+    cast,
+)
 
 binary_types = (bytes, bytearray)
 text_types = (str,)
 string_types = (bytes, str, bytearray)
 
 
-def is_binary(value):
+def is_binary(value: Any) -> bool:
     return isinstance(value, binary_types)
 
 
-def is_text(value):
+def is_text(value: Any) -> bool:
     return isinstance(value, text_types)
 
 
-def is_string(value):
+def is_string(value: Any) -> bool:
     return isinstance(value, string_types)
 
 
-def force_bytes(value, encoding="iso-8859-1"):
+def force_bytes(value: Any, encoding: str = "iso-8859-1") -> bytes:
```

Comment:
Maybe it's worth using `value: Union[str, bytes, bytearray]`, explicitly specifying its type for stricter typing?

Reply:
Yes, thank you! Applied.